mlkem/asm/clean/rej_uniform_asm.s

```diff
@@ -0,0 +1,369 @@
+// SPDX-License-Identifier: Apache-2.0
+
+/*************************************************
+* Name:        simd_memcpy
```

Comment:
I am skeptical that we should write this by hand -- compilers will utilize have a highly-optimized `memcpy` already. What's your motivation here?

Reply:
I couldn't make it compile and link to external memcpy. 
I tried `extern memcpy`, still, the compile won't make it compile. I stuck for a while and decided to write my version. 
Since the data is small, ~600 bytes, an indepdent memcpy could save some code space. 
Is there a way to ask compiler `extern memcpy`?

Reply:
I haven't yet read the core assembly below in detail but am surprised that we need a `memcpy` at all, actually.

Reply:
Honestly, I don't want to write my own version of `memcpy`. I spent so long and I can't figure out why `extern memcpy` or `.extern memcpy` doesn't compile. If I find a way to link `memcpy` I would definitely remove my own.

Reply:
okay, I figure it out, turn out on macos the symbol for `memcpy` is `_memcpy`, and linux is `memcpy`. 
I added a macro for to switch. Now it's worked! the `simd_memcpy` is removed. 
Commit history is cleaned.